### PR TITLE
replace DCAT-link

### PR DESCRIPTION
**Pull request by gabimartini**
This PR replace the DCAT link

### CODE REVIEW

File: src/views/learn/main.njk [hunk 1]
```diff
@@ -39,7 +39,7 @@
         <li style="margin-bottom:25px;"><a class="govuk-link" href="/learn/articles/metadata-model">Metadata model</a></li>
         <li style="margin-bottom:25px;"><a class="govuk-link" href="/learn/articles/publish-data-descriptions-questions">Publish data descriptions questions</a></li>
         <li style="margin-bottom:25px;"><a class="govuk-link" href="/learn/articles/esda">DRAFT Record key information about Essential Shared Data Assets</a></li>
-        <li style="margin-bottom:45px;"><a class="govuk-link" href="/learn/articles/dcat">DRAFT Using metadata to describe data assets in a data catalogue</a></li>
+        <li style="margin-bottom:45px;"><a class="govuk-link" href="https://www.gov.uk/government/publications/recommended-open-standards-for-government/using-metadata-to-describe-data-assets-in-a-data-catalogue">DRAFT Using metadata to describe data assets in a data catalogue</a></li>
```
Comment (kjoshi) [inline src/views/learn/main.njk:42]: Thanks Gabi!
Alasdair has suggested getting rid of the `DRAFT` from the link text.

```suggestion
        <li style="margin-bottom:45px;"><a class="govuk-link" href="https://www.gov.uk/government/publications/recommended-open-standards-for-government/using-metadata-to-describe-data-assets-in-a-data-catalogue">Using metadata to describe data assets in a data catalogue</a></li>
```

Comment (duncanheron) [reply src/views/learn/main.njk:42]: Should this open in a new window as its leaving the site?

Comment (gabimartini) [reply src/views/learn/main.njk]: @kjoshi done
@duncanheron none of the other links that are sending to outside the site we are opening new window, I kept in the same way. But I'm happy to make it open new window if you think it's better in this case

Comment (kjoshi) [reply src/views/learn/main.njk]: I was thinking the same @duncanheron, but if none of the other external links open in a new tab I guess it's fine to let this one be the same?

Comment (duncanheron) [reply src/views/learn/main.njk]: if its consitenly bad, then ok 😄

**Review by kjoshi — APPROVED** [review]
👍

**Pull request merged**